fcrepo-webapp/src/main/webapp/static/constraints/ExternalMessageBodyException.rdf

```diff
@@ -17,10 +17,10 @@
     <owl:Ontology rdf:about="static/constraints/ExternalMessageBodyException">
         <rdfs:label xml:lang="en">External Message Body Constraint</rdfs:label>
         <rdfs:comment xml:lang="en">Part of the request for creating/updating an External Message Body was missing
-            or malformed. Link header should include a URI reference to the external resource, a link relation
-            type ('rel') that's set to "http://fedora.info/definitions/fcrepo#ExternalContent";',
-            a 'handling' attribute set to one of these three things: proxy, redirect, or copy, and an optional
-            attribute of 'type' representing the content type of the external body.
+            or malformed. Link header must include a URI reference to an external resource from an allowed path,
+            a link relation type ('rel') that is set to "http://fedora.info/definitions/fcrepo#ExternalContent";',
```

Comment:
I think you can take out the `;'` at the end of the line.  Looks like a typo that was there before you updated this code.